edison/static/css/login-signin.css

```diff
@@ -0,0 +1,94 @@
+body{
+    background-color: rgb(116,116,116);
+    background-image: url(../Pictures/login/background.jpg);
+}
+
+.container-background{
+    background-color: rgba(0,0,0,0);
+    filter: blur(0px);
+}
+
+.container-padding{
+    padding: 50px;
+}
+
+.switch-sign-up-block{
+    display:block;
+    padding: 0;
+    padding-top: 15px;
+    padding-bottom: 15px;
+}
+
+.switch-sign-up-text-block{
+    padding: 25px;
+    background-color: rgba(0,0,0,0.7);
+    padding-top: 50px;
+}
+
+.login-block-column{
+    display:block;
+    background-color: rgba(141, 64, 64, 0);
+    padding: 0;
+}
+
+.login-block{
+    background-color: #ffffff;
+    position: absolute;
+    padding: 20px;
```

Comment:
Should be removed, you defined each padding side underneath.
Also you could shorten padding definitions like so => padding: top right bottom left.
So it would look like this => padding: 40px 30px 40px 30px

Reply:
ack